pkg/shell/index.html

```diff
@@ -8,7 +8,10 @@
     <link href="../../static/branding.css" rel="stylesheet">
     <script src="../base1/cockpit.js"></script>
     <script src="../manifests.js"></script>
+    <!-- HACK: C bridge loads translations via glob and Python via manifest.js -->
+    <script src="../*/po.manifest.js"></script>
     <script src="../*/po.js"></script>
+    <script src="po.js"></script>
```

Comment:
Of course!  Now that the shell no longer gets its own translations sent via manifests.js, you need to explicitly request them.  Subtle!

Reply:
Yup :)